clients/rkecli/state.go

```diff
@@ -129,7 +128,7 @@ func NewClusterFile(state *cluster.FullState, dirName string, config *Config) (c
 			rkeConfig.Nodes[i].SSHKeyPath = appendSSHPath(rkeConfig.Nodes[i].SSHKeyPath, config.SSHPath)
 		}
 	} else {
-		return "", errors.Wrap(err, "rke SSHPath or SSHKey not configured")
+		return "", errors.Wrap(errors.New("missing SSH configuration"), "rke SSHPath or SSHKey not configured")
```

Comment:
What do you think about removing the wrap since the first part is always nil? Something like this?

```suggestion
		return "", errors.New("missing SSH configuration")
```